specs/permissions.md

```diff
@@ -156,6 +158,9 @@ The "userConsentDescription" member is a REQUIRED string that describes the perm
 ### requiresAdminConsent
 The "requiresAdminConsent" member is a boolean value with a default value of false. When true, this permission can only be consented by an adminstrator.
 
+### isPreauthorizationOnly
+The "isPreauthorizationOnly" member is a boolean value with a default value of false. When true, the permission is preauthorized by the resource owner and does not prompt the user to consent.
```

Comment:
Is this property applicable to appOnly schemes?

Reply:
Yes, I have checked with Jeff and is appOnly schemes.